docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -506,6 +506,9 @@ Due to that we can't see into the future, kubeadm CLI vX.Y may or may not be abl
 Example: kubeadm v1.8 can deploy both v1.7 and v1.8 clusters and upgrade v1.7 kubeadm-created clusters to
 v1.8.
 
+Please also check our [installation guide](/docs/setup/independent/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl)
+for more information on the version skew between kubelets and the control panel.
+
```

Comment:
Is "control panel" correct? Or is it "control plane"?

Reply:
Seems like it should be `control plane`.
BTW, sorry for the incorrect review comment in https://github.com/kubernetes/kubernetes.github.io/pull/5815#discussion_r143420140 (`s/pabel/panel`)

Reply:
Good catch! I just corrected